charts/golang/templates/sa.yaml

```diff
@@ -2,3 +2,7 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: {{ include "common.names.fullname" . }}
+  {{- if .Values.serviceAccountAnnotations }}
+  annotations:
+    {{- include "common.tplvalues.render" (dict "value" .Values.serviceAccountAnnotations "context" $) | nindent 4 }}
+  {{- end }}
```

Comment:
While we are in here can we add the common annotations that are present in the deployment and other manifests?